src/main/java/pl/isa/javaers/Main.java

```diff
@@ -2,6 +2,7 @@
 
 public class Main {
     public static void main(String[] args) {
-
+        GeneratorAlertID.generatorAlertID();
```

Comment:
Czy to w mainie jest potrzebne, czy tylko dla testów?

Reply:
Tylko dla testu